lib/heroku/command/pg.rb

```diff
@@ -14,6 +14,16 @@ def set_commands(shorthand)
     end
   end
 
+  # pg:stats_reset
+  # calls the Postgres functions pg_stat_reset()
```

Comment:
Yes, we absolutely should. Would you be able to help me with that a bit later? I'm not as clear on what the *other* stats reset function does and want to be sure I'm not misleading anyone with bad info.